package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "watson-openwhisk-sdk",
-  "version": "0.5.2",
+  "version": "0.8.1",
```

Comment:
Why are we going from `0.5.2` to `0.8.1` ? Is that arbitrary?

Reply:
The version in the package.json wasn't updated in a long time; it was overlooked by accident since that version is only used when running tests locally - past releases were 0.6.2, 0.7.0 (release 7 which was not published to UI), and 0.8.0 (in different branch) so this is 0.8.1 to differentiate

Reply:
version going to .8 as it adds new features vs a patch release

Reply:
Okay that's fine